test/preflight/apps_v2_integration_test.go

```diff
@@ -455,16 +456,27 @@ func TestErrOutput(t *testing.T) {
 	res = f.FlyAllowExitFailure("machine update --vm-memory 10 %s --yes", firstMachine.ID)
 	require.Contains(f, res.StdErrString(), "invalid memory size")
 
-	f.Fly("machine update --vm-cpus 4 %s --vm-memory 2048 --yes", firstMachine.ID)
+	// This should fail on GPU machines because they're performance VMs.
+	if f.IsGpuMachine() {
+		res = f.FlyAllowExitFailure("machine update --vm-cpus 4 %s --vm-memory 2048 --yes", firstMachine.ID)
+		require.Contains(f, res.StdErrString(), "memory size for config is too low")
```

Comment:
Nit: You can `return` from here since the rest is having `if ! f.isGpuMachine()`